src/public/js/components/categoryComponents/sidebar.js

```diff
@@ -0,0 +1,200 @@
+import { cleanParam, updateURL } from './queryParams';
+
+const sidebar = document.querySelector('.sidebar__nav');
+const sidebarOverlay = document.querySelector('.sidebar__background');
+const sidebarClose = document.querySelector('#sidebar-close');
+//const sidebarOpen = document.querySelector('#sidebar-open');
+const sidebarSections = document.querySelectorAll('.menu-section');
+const clearFiltersBtn = document.querySelector('#clear-filters-btn');
+const globalSelectedFiltersContainer = document.querySelector('.selected-filters__summary');
+let countFiltersApplied = 0;
+const tags = {};
+
+function addTag(param, value) {
+  tags[`${param}`] = [...(tags[`${param}`] ?? []), value];
+}
+
+function deleteTag(param, value) {
+  tags[`${param}`] = (tags[`${param}`] ?? []).filter((element) => element !== value);
```

Comment:
It's unnecessary that you use template literals to access a one element in the objet/array
```suggestion
  tags[param] = (tags[param] ?? []).filter((element) => element !== value);
```